rules/default.js

```diff
@@ -5,26 +7,23 @@ module.exports = function (options) {
     // Block level
 
     code : function (code, lang, escaped) {
-        var result;
-
-        // VERY STRANGE CODE, WHY DO WE PROCESS 'javascript' AS IS ?
-        if ('javascript' == lang) {
```

Comment:
Purpose of `code` function to output code sample in markdown. If you want to insert plain `BEMJSON` to output and then process it you need another solution.